app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split(" ")) == 3:
+        command, file, new_file = command.split(" ")
```

Comment:
It would be better to not override the `command` variable that comes to the function. You could allocate another variable for the command keyword, e.g. `command_keyword`.